x-pack/plugins/monitoring/server/kibana_monitoring/collectors/get_kibana_usage_collector.js

```diff
@@ -23,7 +23,7 @@ export function getKibanaUsageCollector(server) {
   const { collectorSet } = server.usage;
   return collectorSet.makeUsageCollector({
     type: KIBANA_USAGE_TYPE,
-    async fetch(callCluster) {
+    async fetch({ callCluster }) {
```

Comment:
Next step will be to make this collector use the saved objects client, which will need to have a method to do the summarization being done in this file. I left out that change to keep the PR small.